modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -560,11 +565,17 @@ object ObservationService {
           )
         }
 
+        def itcValidations(params: GeneratorParams): F[ObservationValidationMap] =
+          itcService(itcClient).selectOne(pid, oid, params).map:
+            // N.B. there will soon be more cases here
+            case Some(_) => ObservationValidationMap.empty
+            case None => ObservationValidationMap.empty.add(ObservationValidation.itc("ITC results are not present."))
```

Comment:
I think we discussed this, but just in case there may be no results yet simply because we haven't requested them.